packages/registry/registry/polkadot-ui/blocks/address-input/address-input.dedot.tsx

```
@@ -23,7 +23,8 @@ function AddressInputInner(props: AddressInputProps) {
 
   const services = useMemo(
     () => ({
-      useIdentity,
+      useIdentityOf: (address: string, identityChain?: NetworkId) =>
+        useIdentityOf({ address, chainId: identityChain ?? paseoPeople.id }),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)